tutorial/consuming_packages/getting_started.rst

```diff
@@ -0,0 +1,223 @@
+.. _consuming_packages_getting_started:
+
+Getting started: build a simple CMake project using Conan
+=========================================================
+
+Let's get started with an example: We are going to create a string compressor application
+that uses one of the most popular C++ libraries: `Zlib <https://zlib.net/>`__.
+
+.. important::
+
+    In this example, we will retreive the zlib Conan package from a Conan repository with
+    packages compatible for Conan 2.0. To run this example succesfully you should add this
+    remote to your Conan configuration doing:
+    ``conan remote add conanv2 https://conanv2.jfrog.io/artifactory/api/conan/conan --index 0``
+
+We'll use CMake as build system in this case but keep in mind that Conan **works with any
+build system** and is not limited to using CMake. You can check more examples with other
+build systems in the :ref:`Read More
+section<consuming_packages_getting_started_read_more>`.
+
+.. note::
+
+    The source files to recreate this project are available in the `examples2.0 repository
+    <https://github.com/conan-io/examples2>`_ in GitHub. You can skip the manual
+    creation of the folder and sources with this command:
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/conan-io/examples2.git && cd tutorial/consumer/getting_started
```

Comment:
```suggestion
        $ git clone https://github.com/conan-io/examples2.git
        $ cd tutorial/consumer/getting_started
```
Better if they familiarize with the folder structure of the repo, and shorter cmd lines are copied better too